docs/src/shared/PreviewComponentWithCode.tsx

```diff
@@ -35,15 +34,6 @@ const PreviewComponentWithCode: React.FC<PreviewComponentProps> = (props): JSX.E
                 onMouseLeave={(): void => setShow(false)}
             >
                 <CodeBlock code={code} language="jsx" />
-                <Box
-                    sx={{
-                        position: 'absolute',
-                        bottom: '16px',
-                        right: '400px',
-                    }}
-                >
-                    {show && <CopyToClipboard title={'Copy All'} copyText={code} />}
-                </Box>
```

Comment:
You could have left your button row in here and passed the copytext and url through here and only render the button row if one of those 2 props are available. That would save you a bunch of copy paste.